lib/modules/platform/gerrit/scm.ts

```diff
@@ -129,19 +129,16 @@ export class GerritScm extends DefaultGitScm {
         hasChanges = await git.hasDiff('HEAD', 'FETCH_HEAD'); //avoid empty patchsets
       }
       if (hasChanges || commit.force) {
+        const pushOptions = ['notify=NONE'];
+        if (commit.autoApprove) {
+          pushOptions.push('l=Code-Review+2');
```

Comment:
is this a faster ways for `client.approveChange` ?

Reply:
Not only faster, but it also saves 2 API calls to Gerrit on each run.